exercises/Cielo-Raymundo/short-response.md

```diff
@@ -0,0 +1,21 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+    Function parameters are "placeholders" that go in the function that have no value, but are being used. Arguments are actual values that go 
+    in the function 
+
+2. What happens when you invoke a JavaScript function with a missing argument? What about calling a function with _extra_ arguments? 
+    If you call a function with a missing argument then it will, not run and get an error. If you call a function with extra arguments, then
```

Comment:
Yes for the second part. extra arguments are ignored. However, missing arguments are passed as `undefined`.